internal/sandbox/sandbox.go

```diff
@@ -403,11 +421,17 @@ func (s *podmanSandbox) init() error {
 		return fmt.Errorf("error creating container: %w", err)
 	}
 
-	if args := s.copyArgs(); len(args) > 0 {
-		if err := podmanRun(args...); err != nil {
-			return fmt.Errorf("failed copying arguments into sandbox: %w", err)
+	// run each copy command separately
+	for _, copyOp := range s.copies {
+		copyArgs := copyOp.args(s.container)
+		log.Info("podman copy: " + copyOp.String())
+		if err := podmanRun(copyArgs...); err != nil {
+			exitErr := err.(*exec.ExitError)
+			log.Debug("copy into sandbox failed", "args", copyArgs, "stderr", exitErr.Stderr)
```

Comment:
Since we return the error and log it later on I believe we don't have to do a log.Debug here.

Reply:
Ah thanks, I forgot to remove this after testing